src/it/resources/data/typicalDataBundle.json

```diff
@@ -48,6 +54,43 @@
       "name": "Typical Course 2",
       "institute": "TEAMMATES Test Institute 1",
       "timeZone": "Asia/Singapore"
+    },
+    "typicalCourse1": {
+      "createdAt": "2012-04-01T23:58:00Z",
+      "id": "idOfTypicalCourse1",
+      "name": "Typical Course 1",
+      "institute": "TEAMMATES Test Institute 1",
+      "timeZone": "Africa/Johannesburg"
+    },
+    "typicalCourse2": {
+      "createdAt": "2012-04-01T23:59:00Z",
+      "id": "idOfTypicalCourse2",
+      "name": "Typical Course 2",
+      "institute": "TEAMMATES Test Institute 1",
+      "timeZone": "UTC"
+    },
+    "typicalCourse3": {
+      "createdAt": "2012-04-02T23:58:00Z",
+      "deletedAt": "2012-04-12T23:58:00Z",
+      "id": "idOfTypicalCourse3",
+      "name": "Typical Course 3",
+      "institute": "TEAMMATES Test Institute 1",
+      "timeZone": "UTC"
+    },
+    "typicalCourse4": {
+      "createdAt": "2012-04-02T23:58:00Z",
+      "id": "idOfTypicalCourse4",
+      "name": "Typical Course 4",
+      "institute": "TEAMMATES Test Institute 1",
+      "timeZone": "UTC"
+    },
+    "typicalCourse5": {
+      "createdAt": "2012-04-02T23:58:00Z",
+      "deletedAt": "2012-04-12T23:58:00Z",
+      "id": "idOfTypicalCourse5",
+      "name": "Typical Course 5",
+      "institute": "TEAMMATES Test Institute 1",
+      "timeZone": "UTC"
```

Comment:
Please do not add so much data to the typical data bundle. We are trying to keep the bundle small. If you need the data for a test, please create a separate data bundle.